src/scanners/common/createRegionalScanner.ts

```diff
@@ -30,11 +33,14 @@ async function scanRegion<T extends ResourceDescription>(
 		// Perform scan
 		const resources = await scanFunction(credentials, rateLimiter, region)
 
+		// Fetch tags
+		const tags = await fetchTags(Object.keys(resources), rateLimiter)
```

Comment:
I see two points here:
- We might also pass the `credentials` to `fetchTags` and use it to initiate the client.
- Having a totally separate rate limiter for the entire `ResourceGroupsTagingAPIClient` can be a good idea, since it has its own request quotas separate from the services client quotas.